lib/alerts/course_alert_manager.rb

```diff
@@ -28,12 +30,17 @@ def self.generate_course_alerts
 
   def initialize
     @courses_to_check = Course.strictly_current.where(withdrawn: false)
+    @tagged_courses_to_check = Tag.courses_tagged_with('ta_support')
```

Comment:
Because this set of courses is specific to the new feature rather than used across multiple alert managers, I think this is better done within the manager. Also, you could scope this with `.current_and_future` to ensure that only ongoing courses that aren't already over can get the alerts.